# Generated for NaoQi method ALBluetooth.getConnectedDeviceList
# list all connected bluetooth devices

---
# a list of connected bluetooth device <name>
string[] value


